pkgs/top-level/python-packages.nix

```diff
@@ -5176,6 +5176,12 @@ self: super: with self; {
 
   incremental = callPackage ../development/python-modules/incremental { };
 
+  indexed_bzip2 = callPackage ../development/python-modules/indexed_bzip2 { };
```

Comment:
```suggestion
  indexed-bzip2 = callPackage ../development/python-modules/indexed-bzip2 { };
```

Reply:
Note while the paths usually use `-` and not `_`, it is common to use `_` in `pname`, since `fetchPypi` inherits from it. ;)